packages/nextjs/app/faqs/page.tsx

```diff
@@ -42,9 +64,41 @@ const faqs = [
   {
     id: 7,
     question: "Is there a leaderboard?",
-    answer:
-      "There isn't a leaderboard for this version of the game. Instead, there is a Stats page which can be found at",
-    link: "/stats",
+    answer: (
+      <>
+        There isn&apos;t a leaderboard for this version of the game. Instead, there is a Stats page which can be found
+        at{" "}
+        <a className="text-primary link break-all lg:break-normal" href="/stats" target="_blank">
+          /stats
+        </a>
+      </>
+    ),
+  },
+  {
+    id: 8,
+    question: "What are the Seasons?",
+    answer: (
+      <>
+        Solidity Invaders is organized into seasons, each featuring a new set of challenges. Season 1 (Bangkok) includes
+        the 12 challenges from{" "}
+        <a
+          className="text-primary link break-all lg:break-normal"
+          href="https://ctf-devcon.buidlguidl.com/"
+          target="_blank"
+        >
+          Devcon SEA
+        </a>
+        . Season 2 (Buenos Aires) will include 12 challenges played at{" "}
```

Comment:
I think we can remove "will" since we will publish this branch when the new challenges are online.